src/x11.rs

```diff
@@ -294,13 +304,21 @@ impl<'a> BufferImpl<'a> {
         unsafe { self.0.buffer.buffer_mut() }
     }
 
+    pub fn age(&self) -> Option<NonZeroU8> {
```

Comment:
I'd suggest to use `0` actually, since it'll naturally work. Like we have 0 in buffer_age ext.

Reply:
Yeah, after thinking about it, that is probably simpler.